Source/Plugins/EditorModules/ObjectInspector/Modules/ObjectInspector.cs

```diff
@@ -28,7 +23,13 @@ public partial class ObjectInspector : DockContent
 		private ObjectSelection.Category displayCat          = ObjectSelection.Category.None;
 
 		private ExpandState              gridExpandState     = new ExpandState();
-
+		private ObjectInspectorState userSettings;
+		
+		public ObjectInspectorState UserSettings
+		{
+			get { return this.userSettings; }
+			set { this.userSettings = value; }
+		}
```

Comment:
Style: This property should follow the grouping of the others, i.e. not have a newline following it.